obs-studio-client/source/controller.cpp

```diff
@@ -237,80 +282,23 @@ std::shared_ptr<ipc::client> Controller::host(std::string uri) {
 	if (m_isServer)
 		return nullptr;
 
-	// Store info
-	std::string program = serverBinaryPath;
 	std::string commandLine = '"' + serverBinaryPath + '"' + " " + uri;
```

Comment:
This is slow and results in four separate re-allocations, consider also improving this with std::stringstream or a sprintf_s.